### PR TITLE
This is really about forks, not LTS per se.

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: library-best-practices/JLBP-9.md [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+This can also lead to diamond dependency conflicts where no version works for
+all consumers. 
 
-As an alternative, if you need to support both consumers on the old version of
-Java and consumers wanting features provided by a newer version of Java, a
-long-term support (LTS) release can be created to continue supporting the old
-Java version, and a new major version of the library can be released with a
-higher minimum Java version. The Java package and maven ID should be kept the
-same in the new major version.
+Thus, libraries should wait a reasonable period of time
```
Comment (suztomo) [inline library-best-practices/JLBP-9.md]:
```suggestion
Thus, library mainteiners should wait a reasonable period of time
```

Anthropomorphism

File: library-best-practices/JLBP-9.md [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-version. This can result in a situation where the ecosystem is stuck on older
-library versions, preventing the pickup of important things like security fixes,
-and it can also lead to diamond dependency conflicts where no version works for
-all consumers. Thus, libraries should make sure their actively-maintained
-consumers advance their minimum version of Java before they advance it
-themselves. (Waiting for every last consumer could block an upgrade permanently,
-due to cases like abandoned projects.)
+Imagine library B depends on library A and both work with Java 8.
+If library A releases a new version that requires Java 11, library B cannot upgrade
+unless it also starts requiring Java 11.
```
Comment (suztomo) [inline library-best-practices/JLBP-9.md]:
```suggestion
the dependency unless it also starts requiring Java 11. 
```

Comment (elharo) [reply library-best-practices/JLBP-9.md]: Unfortunately suggesting github changes interferes with the CLA bot.

Comment (suztomo) [reply library-best-practices/JLBP-9.md]: Yes, I remember that. I don't expect you to click "accept" for these suggestions.

Comment (elharo) [reply library-best-practices/JLBP-9.md]: rewritten

File: library-best-practices/JLBP-9.md [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
-all consumers. Thus, libraries should make sure their actively-maintained
-consumers advance their minimum version of Java before they advance it
-themselves. (Waiting for every last consumer could block an upgrade permanently,
-due to cases like abandoned projects.)
+Imagine library B depends on library A and both work with Java 8.
+If library A releases a new version that requires Java 11, library B cannot upgrade
+unless it also starts requiring Java 11. 
+If library B is not able to do that—for instance because it must work in an environment
+such as App Engine that does not yet support Java 11—it must keep using the older
+version of library B, missing out on security and bug fixes in the newer version.
```
[the remark this answers is not in the document]
Comment (elharo) [reply library-best-practices/JLBP-9.md]: done

File: library-best-practices/JLBP-9.md (hunk 1 above)
Comment (elharo) [reply library-best-practices/JLBP-9.md]: rewritten

File: library-best-practices/JLBP-9.md [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
+actively-maintained dependents already require that version of Java.
+It is rarely possible to wait for all dependents to upgrade.
+Some projects are abandoned or lightly maintained and may never upgrade.
+
+As an alternative, to support both dependents on the old version of
+Java and dependents wanting features provided by a newer version of Java, 
+you can continue development in two forks, version N for the older version of
+Java and version N+1 for the newer version of Java. Typically in this case,
+the major version is incremented to distinguish the two versions.
+The Java package and Maven coordinates should be the same in the new major version.
```
Comment (suztomo) [inline library-best-practices/JLBP-9.md]: For the background of the last sentence, you may want to add reference to JLBP-6 "Package and artifact renaming rules".

File: library-best-practices/JLBP-9.md [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
-all consumers. Thus, libraries should make sure their actively-maintained
-consumers advance their minimum version of Java before they advance it
-themselves. (Waiting for every last consumer could block an upgrade permanently,
-due to cases like abandoned projects.)
+Imagine that library B depends on library A and both work with Java 8.
+If library A releases a new version that requires Java 11, library B cannot upgrade
+to the new version of library A unless it also starts requiring Java 11. 
+If library B is not able to do that—for instance because it must work in an environment
+such as App Engine that does not yet support Java 11—it must keep using the older
+version of library A, missing out on security and bug fixes in the newer version.
```
Comment (suztomo) [inline library-best-practices/JLBP-9.md]: Just a suggestion.

```suggestion
version of library A, missing out on security and bug fixes in its newer versions.
```

to emphasize the effect is not just one version but other releases in future.

**Pull request merged**